RecogScript.py

```diff
@@ -40,30 +37,36 @@ def check_encodings(all_encodings, all_ids, small_frame, temp_filename="images/t
                 cv2.imwrite(temp_filename, small_frame)
                 image = cv2.imread(temp_filename)
 
-                if not detect_blurry_image(image):
+                if not detect_blurry_image(image, 200):
 
                     frame = cv2.resize(image, (0, 0), fx=0.75, fy=0.75)
                     unknown_encodings, unknown_images = get_images_and_encodings(frame)
 
                     i = 0
                     for encoding in unknown_encodings:
-                        encoded_image = base64.b64encode(unknown_images[i]).decode('utf-8')
+                        encoded_image = base64.b64encode(np.array(unknown_images[i])).decode('utf-8')
                         encoding_str = str(encoding[i])
                         data = {"img":encoded_image, "npy": encoding_str}
                         add_unknown_person(data)
                         i = i + 1
 
 
+
 '''
 Main script function
 '''
 def main():
-    video_capture = get_camera_ip_from_file("camera_ip.txt")
+    video_capture = []
+    video_capture += get_camera_ip_from_file("camera_ip.txt")
+    x = 0
 
     while True:
-        encodings, all_ids, small_frame = process_video_to_encode(video_capture)
-        check_encodings(encodings, all_ids, small_frame)
+        for v in video_capture:
+            encodings, all_ids, small_frame = process_video_to_encode(v)
+            temp_file_name = "images/temp_%d.jpeg" % x
+            x += 1
```

Comment:
is it necessary to create new temp files everytime? wouldn't it be okay to overwrite the same temp name? because after we turn the temp into  frame  we get encodings and images and store that so we don't necessarily need to store the temp images

Reply:
i think that once we get the "nickname" thing working for the cameras we can do temp_nickname and that'll be fine since they'll get overwritten but for now, that's how i wanted to make sure that it actually worked